markhor_slam/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(markhor_slam)
+
+find_package(catkin REQUIRED)
+
+catkin_package(
+#  INCLUDE_DIRS include
```

Comment:
Since this package is quite empty, I think it's alright to remove the comments inside the `catkin_package` function.